clarity/src/vm/variables.rs

```diff
@@ -123,6 +141,16 @@ pub fn lookup_reserved_variable(
                 let chain_id = env.global_context.chain_id;
                 Ok(Some(Value::UInt(chain_id.into())))
             }
+            NativeVariables::StacksBlockHeight => {
```

Comment:
These need to be gated

Reply:
I believe this is already gated by the `lookup_by_name_at_version` that is the source of the `variable`.